### ISSUE TITLE
Using --ssl-native calls non-terminating `openssl.Linux.x86_64 s_client -ssl2…`

### ISSUE THREAD

**Issue by dilyanpalauzov** [drwetter/testssl.sh#1149]
Calling `./testssl.sh -t imap -p --ssl-native mail.aegee.org:143` calls `./bin/openssl.Linux.x86_64 s_client -state -ssl2 -starttls imap -connect 144.76.142.78 143 -no_comp` and this never terminates.  So the program hangs.

**Comment by drwetter** [comment]
yeah I know. Given the fact that it seems only to occur when using STARTTLS && some special servers && the default run is not using openssl anymore && it seems quite an effort to fix it properly: likely this won't be fixed.

You may use ``--openssl-timeout`` -- the only catch was that this seems to hit another problem (MAX_OSSL_FAIL). Fix follows for the latter

**Comment by drwetter** [comment]
see 2a65bb8

**Comment by dilyanpalauzov** [comment]
If the latter is fixed, why doesn’t `--ssl-native` imply `--openssl-timeout` with some reasonable default?

**Comment by drwetter** [comment]
Because those are two different options which I believe should the users set at their convenience.
Also it doesn' t hang on every host. Your's is an exception.
